foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/logging/LogCategory.java

```diff
@@ -70,8 +71,9 @@
     SEQUENCING( (byte)0x12, SessionLog.SEQUENCING),
     SERVER(     (byte)0x13, SessionLog.SERVER),
     SQL(        (byte)0x14, SessionLog.SQL),
-    TRANSACTION((byte)0x15, SessionLog.TRANSACTION),
-    WEAVER(     (byte)0x16, SessionLog.WEAVER);
+    THREAD(     (byte)0x15, SessionLog.THREAD),
```

Comment:
do not change existing constants, add a new one instead - consider backwards compatibility